functions/github/github.js

```diff
@@ -247,12 +248,29 @@ async function getBuildArtifactsWorkflow(octokit, releaseBranchName) {
   }
 }
 
+/**
+  * Verifies the signature of a request.
```

Comment:
What's the signature for though? I know what you mean, but this is an example of a situation where the docs need to be a bit more specific. Docs are not only for your future self, but people completely unfamiliar with the project. 

What is this signature, why do we need to verify it, and what does it have to do with GH webhooks?